packages/forms-web-app/src/routes/pdf/index.js

```diff
@@ -0,0 +1,39 @@
+const axios = require('axios');
+const FormData = require('form-data');
+const express = require('express');
+const { getExistingAppeal } = require('../../lib/appeals-api-wrapper');
+const { getDepartmentFromId } = require('../../services/department.service');
+const { VIEW } = require('../../lib/views');
+
+const router = express.Router();
+
+router.get('/source/:id', async (req, res) => {
```

Comment:
hacked together based on the confirm your answers stuff + following the middleware router to see what it did

this isn't 'safe' but it gets data given a good id.